src/software/ai/hl/stp/action/move_action.cpp

```diff
@@ -23,6 +22,7 @@ std::unique_ptr<Intent> MoveAction::updateStateAndGetNextIntent(
     this->enable_dribbler   = enable_dribbler;
     this->move_type         = move_type;
     this->autokick          = autokick;
+    this->ball_navigation   = ball_navigation;
```

Comment:
personal preference, but maybe `ball_navigation_type` is a bit more descriptive (or maybe `ball_collision_type` with Akhil's suggestion)? Don't have any super strong rationale though, so entirely your call.

Reply:
+1 for this slightly more descriptive naming of some kind would be good

Reply:
done